instrumentation/github.com/bradfitz/gomemcache/example/client.go

```diff
@@ -0,0 +1,90 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/bradfitz/gomemcache/memcache"
+
+	otelgomemcache "go.opentelemetry.io/contrib/instrumentation/github.com/bradfitz/gomemcache"
+	otelglobal "go.opentelemetry.io/otel/api/global"
+
+	oteltracestdout "go.opentelemetry.io/otel/exporters/stdout"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+func main() {
+	var host, port = os.Getenv("HOST"), "11211"
+
+	initTracer()
+	ctx := context.Background()
+
+	c := otelgomemcache.NewClientWithTracing(
+		memcache.New(
+			host + ":" + port,
+		),
+	)
+
+	t := otelglobal.Tracer("memcached-test")
```

Comment:
This variable `t` is not used. I would suggest that the call to `Tracer()` belongs inside the call to `NewClientWithTracing()`, which should accept a `trace.Provider` and allocate its own `Tracer` using the name of this library and this library's semver.

Reply:
Looks like it is used on the next line (wondering if that came after this comment). I think the intent here is to have the created span below act as a parent for all the memcached span, right @matej-g ?